src/30_Advanced/Amp-Carousel_with_Amp-List.html

```diff
@@ -0,0 +1,94 @@
+<!---{
+  "preview": "default"
+  }--->
+<!--
+  #### Introduction
+  AMP users might want to pull data from an endpoint to display in a carousel. This example incoporates a dynamic amp-carousel as well as showing how to not only rely on the images served from the endpoint
```

Comment:
Can you explain this part: 'as well as showing how to not only rely on the images served from the endpoint' a bit more? How relevant is it to the sample?

Reply:
We are using AMP with C# for e-commerce, so the way we are using it:

src="<%= Model.Media.PrimaryHeroImage %>  [src]="skuState[skuState.currentSkuState].HeroImage". 

Depending on if a user has selected a different color/sku, we want that image to be the first image in our carousel.